src/app/profile/[path]/edit-profile.tsx

```
@@ -198,33 +198,33 @@ export default function EditProfile({ userRole, fhirId }: Props) {
         break;
       case 'email':
         if (!value) {
-          error = 'Email tidak boleh kosong';
+          error = 'Email cannot be empty';
         } else if (!validateEmail(value)) {
-          error = 'Format email tidak valid';
+          error = 'Email format is invalid';
         }
         break;
       case 'phone':
         const phoneRegex = /^\+?[0-9]{8,15}$/;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)